internal/web/components/pkg_detail_btns/controller.go

```diff
@@ -23,7 +23,13 @@ func getId(pkgName string) string {
 	return fmt.Sprintf("%v-%v", TemplateId, pkgName)
 }
 
-func Render(w io.Writer, tmpl *template.Template, pkg *v1alpha1.Package, status *client.PackageStatus, manifest *v1alpha1.PackageManifest, updateAvailable bool) error {
```

Comment:
Could you please revert this change and check if the pipelines still pass? Because if the previous version (i.e. the long line) did not cause any errors in the pipeline, then I think you can leave it in the long version.